src/scalar.jl

```diff
@@ -75,14 +76,14 @@ function SciMLBase.solve(prob::NonlinearProblem, ::Bisection, args...; maxiters
   fl, fr = f(left), f(right)
 
   if iszero(fl)
-    return BracketingSolution(left, right, EXACT_SOLUTION_LEFT)
+    return BracketingSolution(left, right, EXACT_SOLUTION_LEFT,fl)
```

Comment:
Do I need to return `SciMLBase.build_solution` here too? Because if we need to do, we need to handle how `u0` as a Tuple will be handled here: https://github.com/SciML/SciMLBase.jl/blob/master/src/solutions/nonlinear_solutions.jl#L26? What should it be in that case? @ChrisRackauckas

Reply:
Maybe change it to `N = length((size(u)...,))` since it should not be able to change over the solution. Yes, it should use `build_solution` like all others, and treat `u` as the `left`.

Reply:
or well, based on EXACT_SOLUTION_LEFT choose `u` as left etc.